src/cryo_tank/tanksize.jl

```diff
@@ -74,7 +75,9 @@ function tanksize!(fuse_tank, z::Float64, Mair::Float64, xftank::Float64,
         Winner_tot, lcyl1, tskin, Rinnertank, Vfuel, Winnertank, Wfuel_tot, Winsul_sum, t_head, Whead, Wcyl, Wstiff, Winsul,
         Sinternal, Shead_insul, l_inner = size_inner_tank(fuse_tank, fuse_tank.t_insul)
 
-        if ("vacuum" in fuse_tank.material_insul) || ("Vacuum" in fuse_tank.material_insul) #If tank is double-walled
+        flag_vacuum = check_vacuum(fuse_tank.material_insul) #check if there is a vacuum layer
```

Comment:
This is interesting, I wonder if insulation materials can be something that extends the `Materials` type in a future refactoring? E.g., something like `ThermalInsulators` like `StructuralAlloys` or something. This is fine for now though!